src/components/Editor/index.js

```diff
@@ -782,6 +762,16 @@ class Editor extends PureComponent {
     };
   }
 
+  renderDebugLine() {
+    return DebugLine({
```

Comment:
This runs into the same thing with selected v visible, but yeah, I think it makes sense to not render if the visible document shouldn't have a highlight.